vnext/Shared/OInstance.cpp

```diff
@@ -378,6 +379,11 @@ InstanceImpl::InstanceImpl(
 #endif
         }
         case JSIEngineOverride::Chakra:
+          // Applies only to ChakraCore-linked binaries.
+          Microsoft::React::SetRuntimeOptionBool("JSI.ForceSystemChakra", true);
```

Comment:
>SetRuntimeOptionBool [](start = 28, length = 20)

Yup, that's better! Thanks!

Technically, ChakraRuntimeHolder already knows about that option inside of m_devSettings, so it could be set at a lower ChakraRuntimeHolder level.

But regardless, passing parameters through global static Key/Values makes code overly complex, and hard to reason about, and thus hard to maintain or refactor in the future. I realize that Chakra is near end of its lifetime, but we should still follow good engineering practices, especially in Open Source projects. Leaving it up to you on decision on how to pass this parameter. Perhaps other folks on this review also have opinions about this.